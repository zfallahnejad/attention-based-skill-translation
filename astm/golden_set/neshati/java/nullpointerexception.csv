22656,nullpointerexception
522444,nullpointerexception
101361,nullpointerexception
992484,nullpointerexception
438154,nullpointerexception
871026,nullpointerexception
1221571,nullpointerexception
1202025,nullpointerexception
571407,nullpointerexception
653856,nullpointerexception
1679863,nullpointerexception
157882,nullpointerexception
335858,nullpointerexception
1380752,nullpointerexception
1707091,nullpointerexception
