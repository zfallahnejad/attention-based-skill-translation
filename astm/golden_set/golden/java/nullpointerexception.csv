22656,51,40,0.5604395604395604
522444,36,38,0.4864864864864865
101361,29,19,0.6041666666666666
992484,24,33,0.42105263157894735
438154,22,19,0.5365853658536586
871026,21,30,0.4117647058823529
1221571,16,12,0.5714285714285714
653856,15,17,0.46875
571407,15,11,0.5769230769230769
1202025,15,4,0.7894736842105263
1679863,14,9,0.6086956521739131
157882,13,5,0.7222222222222222
335858,12,4,0.75
1380752,12,2,0.8571428571428571
1707091,11,11,0.5
367273,10,9,0.5263157894736842
1587046,10,7,0.5882352941176471
